src/nvim/tui/term_input.inl

```diff
@@ -298,3 +286,10 @@ static void term_input_destroy(TermInput *input)
   event_poll(0);  // Run once to remove references to input/timer handles
   xfree(input);
 }
+
+static void term_input_set_encoding(TermInput *input, char* enc)
+{
+  int enc_flag = strcmp(enc, "utf-8") == 0 ? TERMKEY_FLAG_UTF8
+                                           : TERMKEY_FLAG_RAW;
```

Comment:
Or we could just check `enc_utf8` as I mentioned: https://github.com/neovim/neovim/pull/2469#discussion_r29997037